cmd/server/config.go

```diff
@@ -0,0 +1,119 @@
+package main
+
+import (
+	"os"
+	"path/filepath"
+	"text/template"
+
+	"github.com/kubernetes-incubator/cri-o/server"
+	"github.com/opencontainers/runc/libcontainer/selinux"
+	"github.com/urfave/cli"
+)
+
+const (
+	ocidRoot   = "/var/lib/ocid"
+	conmonPath = "/usr/libexec/ocid/conmon"
+	pausePath  = "/usr/libexec/ocid/pause"
+)
+
+var commentedConfigTemplate = template.Must(template.New("config").Parse(`
+# Commented version of ocid configuration file. Generated automatically.
+
+# The "ocid" table contains all of the server options. There are no
+# bare options. "ocid" contains global options that don't really
+# relate to any particular sub-table.
+[ocid]
+
+# root is a path to the "root directory" where all information not
+# explicitly handled by other options will be stored.
+root = "{{ .Root }}"
+
+# sandbox_dir is the directory where ocid will store all of its sandbox
+# state and other information.
+sandbox_dir = "{{ .SandboxDir }}"
+
+# container_dir is the directory where ocid will store all of its
+# container state and other information.
+container_dir = "{{ .ContainerDir }}"
+
+# The "ocid.api" table contains settings for the kubelet/gRPC
+# interface (which is also used by ocic).
+[ocid.api]
+
+# listen is the path to the AF_LOCAL socket on which cri-o will listen.
+# This may support proto://addr formats later, but currently this is just
+# a path.
+listen = "{{ .Listen }}"
+
+# The "ocid.runtime" table contains settings pertaining to the OCI
+# runtime used and options for how to set up and manage the OCI runtime.
+[ocid.runtime]
+
+# runtime is a path to the OCI runtime which ocid will be using. Currently
```

Comment:
I think we should stay with the simple explanation here and perhaps put comments like this in the Release Notes.  If I install the first version of ocid, and modify the config file, it could live for several years, and hopefully comments like this will become false over time.